rma/views/rma_views.xml

```diff
@@ -0,0 +1,269 @@
+                            name="action_lock"
+                            states="received"/>
+                    <button type="object"
+                            string="Unlock"
+                            name="action_unlock"
+                            states="locked"/>
+                    <button type="object"
+                            string="Preview"
+                            name="action_preview"/>
+                    <field name="state" widget="statusbar" statusbar_visible="draft,confirmed,received"/>
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button type="object"
+                                name="action_view_receipt"
+                                string="Receipt"
+                                class="oe_stat_button"
+                                icon="fa-truck"
+                                attrs="{'invisible': [('reception_move_id', '=', False)]}">
+                        </button>
+                        <button type="object"
+                                name="action_view_delivery"
+                                class="oe_stat_button"
+                                icon="fa-truck"
+                                attrs="{'invisible': [('delivery_picking_count', '=', 0)]}">
+                            <field name="delivery_picking_count" widget="statinfo" string="Delivery"/>
+                        </button>
+                        <button type="object"
+                                string="Refund"
+                                name="action_view_refund"
+                                class="oe_stat_button"
+                                icon="fa-pencil-square-o"
+                                attrs="{'invisible': [('refund_id', '=', False)]}">
+                        </button>
+                    </div>
+                    <div class="oe_title">
+                        <h1>
+                            <field name="name" readonly="1"/>
+                        </h1>
+                    </div>
+                    <group>
+                        <group>
+                            <field name="partner_id" widget="res_partner_many2one" context="{'search_default_customer':1, 'show_address': 1, 'show_vat': True}" options="{'always_reload': True}"/>
+                            <field name="partner_invoice_id"/>
+                            <field name="picking_id"
+                                   options="{'no_create': True}"/>
+                            <field name="move_id"
+                                   attrs="{'required': [('picking_id', '!=', False)]}"
+                                   options="{'no_create': True}"/>
+                            <field name="product_id"
+                                   force_save="1"
+                                   attrs="{'readonly': ['|', ('picking_id', '!=', False), ('state', '!=', 'draft')]}"/>
```

Comment:
As soon as picking is selected, we need to select the move, but maybe you can enhance move name because it's a little bit long and user is focusing on product I think.

![image](https://user-images.githubusercontent.com/19529533/83941807-a3c6ef00-a7ee-11ea-9c65-0bf1a4192075.png)

Reply:
This wasn't attended.

Reply:
I have added this in the ROADMAP as an improvement to be made

Reply:
Ok, then. Resolve, please ;)